### PR TITLE
Fixed shellcheck errors in create-adu-import-manifest.sh

### PR DESCRIPTION
**Pull request by nihemstr**

### CODE REVIEW

File: tools/AduCmdlets/create-adu-import-manifest.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ write_verbose() {
         YELLOW="$(tput setaf 11)"
         RESET="$(tput sgr0)"
 
-        printf "${YELLOW}VERBOSE: $1${RESET}\n"
+        printf "%sVERBOSE: %s%s\n" "${YELLOW}" "$1" "${RESET}"
```
Comment (JeffMill) [inline tools/AduCmdlets/create-adu-import-manifest.sh:33]: >printf [](http://example.com/codeflow?start=8&length=6)

i could have used echo here.  Does it complain with echo? e.g.

`echo "${YELLOW}VERBOSE: $1${RESET}"`

File: tools/AduCmdlets/create-adu-import-manifest.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ write_error() {
     RED="$(tput setaf 9)"
     RESET="$(tput sgr0)"
 
-    printf "${RED}ERROR: $1${RESET}\n"
+    printf "%sERROR: %s%s\n" "${RED}" "$1" "${RESET}"
 }
```
Comment (JeffMill) [inline tools/AduCmdlets/create-adu-import-manifest.sh:45]: as above.

File: tools/AduCmdlets/create-adu-import-manifest.sh [hunk 3]
```diff
@@ -174,7 +174,7 @@ while getopts "c:f:hi:n:p:t:v:" OPT; do
 done
 
 # Update files are the arguments without switches.
-shift $(expr $OPTIND - 1)
+shift $(( OPTIND - 1))
 UPDATE_FILES=($@)
```
Comment (JeffMill) [inline tools/AduCmdlets/create-adu-import-manifest.sh:178]: did it not complain that this needs quotes?

**Review by JeffMill — APPROVED** [review]
:shipit:

**Comment by JeffMill** [comment]
approved with comments

**Comment by JeffMill** [comment]
thanks for fixing the shellcheck issues. If I check in bad stuff, it's my responsibility to clean it up, so next time, please assign the bug to me.

**Comment by shiyi-peng** [comment]
For some reason the pipeline run for this was not automatically kicked off, so you might need to manually run it.

**Pull request merged**